src/views/Dashboard/Customer/Home/GeneticData/GeneticAnalysisList.vue

```diff
@@ -0,0 +1,188 @@
+<template lang="pug">
```

Comment:
did you create a new one?

Reply:
yes, idunno you already create this tab page.